tests/sentry/snuba/metrics/test_metrics_layer/test_metrics_enhanced_performance.py

```diff
@@ -1338,6 +1339,118 @@ def test_team_key_transactions_my_teams(self):
             },
         ]
 
+    @freeze_time("2022-09-22 11:07:00")
+    def test_team_key_transaction_as_condition(self):
+        now = timezone.now()
+
+        for idx, (transaction, value) in enumerate(
+            (("foo_transaction", 1), ("bar_transaction", 1), ("baz_transaction", 0.5))
+        ):
+            self.store_metric(
+                org_id=self.organization.id,
+                project_id=self.project.id,
+                type="distribution",
+                name=TransactionMRI.DURATION.value,
+                tags={"transaction": transaction},
+                timestamp=(now - timedelta(minutes=idx)).timestamp(),
+                value=value,
+                use_case_id=UseCaseKey.PERFORMANCE,
+            )
+
+        metrics_query = MetricsQuery(
+            org_id=self.organization.id,
+            project_ids=[self.project.id],
+            select=[
+                MetricField(
+                    op="team_key_transaction",
+                    metric_mri=str(TransactionMRI.DURATION.value),
```

Comment:
it's already string:
```
In [2]: from sentry.snuba.metrics.naming_layer.mri import SessionMRI, TransactionMRI

In [3]: TransactionMRI.DURATION.value
Out[3]: 'd:transactions/duration@millisecond'

In [4]: type(TransactionMRI.DURATION.value)
Out[4]: str

```

Reply:
Yes but linter complains because of types

Reply:
do you mean mypy?